registry/domain/models/organization.py

```diff
@@ -29,10 +29,12 @@ def __init__(self, name, org_id, org_uuid, org_type, description,
         self.description = description
         self.short_description = short_description
         self.url = url
+        self.__duns_no = duns_no
         self.contacts = contacts
         self.assets = assets
         self.metadata_ipfs_hash = metadata_ipfs_hash
-        self.groups = []
+        self.groups = groups
+        self.__addresses = addresses
```

Comment:
Add a validation here if a user passes NOne to make it empty list, to avoid none chekcs